packaging/leapp-el7toel8-deps.spec

```
@@ -2,7 +2,14 @@
 %global  lrdname  leapp-repository-deps-el%{rhel}
 %global  ldname   leapp-deps-el%{rhel}
 
-%define leapp_repo_deps  5
+%if 0%{?rhel} == 8
+    %define lpr_name_src leapp-upgrade-el7toel8-deps
```

If I interpret the intention correctly, shouldn't there be (if we are running on RHEL7, we want the 7->8 path and its dependencies):
```suggestion
%if 0%{?rhel} == 7
    %define lpr_name_src leapp-upgrade-el7toel8-deps
```
Similar check as suggested is performed in `leapp-repository.spec`.

---

@MichalHe no, these packages are expected to be created for the **target** system. The deps packages for the system where the leapp / leapp-repository packages are part of the *main* spec files. It's better described in the doc here: https://leapp.readthedocs.io/en/latest/dependencies-leapp-repository.html
(the doc is expected to be updated consequently)